src/inmanta/moduletool.py

```diff
@@ -528,7 +531,20 @@ def get_modules(self, module: Optional[str] = None) -> List[Module]:
         else:
             return self.get_project(load=True).sorted_modules()
 
-    def create(self, name: str) -> None:
+    def create(self, name: str, v1: bool) -> None:
+        if v1:
+            self._create_v1(name)
+        else:
+            module_dir: str = name
+            if os.path.exists(module_dir):
+                raise Exception(f"Directory {module_dir} already exists")
+            cookiecutter(
+                "https://github.com/inmanta/inmanta-module-template.git",
+                checkout="v2",
```

Comment:
Shouldn't we perform a checkout from master? The intention is to have the V2 template on the master branch after the release, no? It looks like no `v2` branch exist at the moment. Was this functionality tested?

Reply:
I expect we will need the V1 a little longer until we have set up the proper distribution channels for v2 modules.

Reply:
I'd discussed both this and https://github.com/inmanta/inmanta-core/pull/3473#discussion_r752924840 with Wouter yesterday but I'd forgotten to share the conclusions. The main reason we need to keep v1 on master for now is that new v2 modules require a v2 std, which we haven't decided how to distribute yet. The `v2` branch does not exist yet but this is part of inmanta/inmanta-module-template#13.